Change from cm to mm

randallwvr90 opened the pull request:
Changed from cm to mm to follow cocorahs standards.

Commit:
Change from cm to mm

Changed from cm to mm to follow cocorahs standards.

ryho commented on main.go:
```diff
@@ -129,9 +129,9 @@ func RunAJob() error {
-	return fmt.Sprintf("%.2f cm \n", cm)
+func convertInchesToMillimetersText(inches float64) string {
+	mm := inches * mmPerInch
+	return fmt.Sprintf("%.2f mm \n", mm)
```
Change to 1 digit of precision

randallwvr90 replied:
See latest commit.

Commit:
Change to 1 digit of precision

Responding to @ryho comment https://github.com/ryho/MetricRain/pull/2#discussion_r654970043
